setup.py

```diff
@@ -53,7 +53,7 @@
     'invenio-assets>=1.0.0b2',
     'invenio-base>=1.0.0a11',
     'invenio-celery>=1.0.0a4',
-    'invenio-classifier~=1.1.0',
+    'invenio-classifier~=1.0,>=1.1.0',
```

Comment:
`~1.0` is what I wanted as well, but @kaplun preferred the stricter `~1.1`... In a way, it depends if the project is following SemVer or not.